### DISCUSSION TITLE
How do I group GitHub actions updates into single PR?

### DISCUSSION THREAD

**Discussion by parafoxia** [renovatebot/renovate#14578] [Request Help]
I've tried looking around for an already asked question on the matter, but can't find an obvious one. If this has been asked before, apologies.

I'm trying to do something similar to this with GitHub Actions:
![image](https://user-images.githubusercontent.com/43846948/157279103-5fe5f59a-74e1-4522-8462-38d98ef149b1.png)

In this screenshot, (I believe) it would group all Python non-development dependencies into a single PR, but I can't find anything in the docs that suggests this can be done with GitHub Actions.

Is this possible?

**Comment by rarkins** [comment]
`"matchManagers": ["github-actions"]`

**Reply by parafoxia** [reply]
Ah awesome. I'll test that when I'm next working on it.

**Reply by micalevisk** [reply]
yeah, that works if we have the `groupName` entry as well.
